src/main/java/application/csv/WriteCsvToResponse.java

```diff
@@ -0,0 +1,66 @@
+package application.csv;
+
+import application.models.Question;
+import application.models.Survey;
+import com.google.common.collect.Lists;
+import com.opencsv.CSVWriter;
+import com.opencsv.bean.ColumnPositionMappingStrategy;
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.PrintWriter;
+import java.util.Collection;
+
+//@Slf4j lombok!
```

Comment:
What does this comment mean?

Reply:
I was poking around the annotations to check if that it can generate the logger, but forgot to remove it.